<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gravity_model_dockwidget</class>
 <widget class="QDockWidget" name="gravity_model_dockwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dock_widget_contents">
   <widget class="QTabWidget" name="tab_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>406</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="analysis_tab">
     <attribute name="title">
      <string>Анализ</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>351</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vertical_layout">
       <item>
        <widget class="QLabel" name="label_consumer_layer">
         <property name="text">
          <string>Выберите слой потребителей</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsMapLayerComboBox" name="cmbox_consumer_layer">
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
         <property name="placeholderText">
          <string>Слой потребителей</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_consumer_attractiveness_field">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выберите атрибут значимости потребителей&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbox_consumer_attractiveness_field">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="placeholderText">
          <string>Атрибут значимости потребителей</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_site_layer">
         <property name="text">
          <string>Выберите слой поставщиков</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsMapLayerComboBox" name="cmbox_site_layer">
         <property name="placeholderText">
          <string>Слой поставщиков</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_site_attractiveness_field">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выберите атрибут значимости  поставщиков&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbox_site_attractiveness_field">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="placeholderText">
          <string>Атрибут значимости поставщиков</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_alpha">
         <property name="text">
          <string>Степень влияния значимости</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spbox_alpha">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_beta">
         <property name="text">
          <string>Степень влияния расстояния</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spbox_beta">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_distance_limit">
         <property name="text">
          <string>Максимальное расстояние до поставщика в метрах</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spbox_distance_limit_meters">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="value">
          <number>9999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ok">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>ОК</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="diagram_tab">
     <attribute name="title">
      <string>Диаграмма</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>299</y>
        <width>371</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontal_layout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_field">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Подписи:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_field">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_export">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Экспорт</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="diagram_layout"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
